canonical-websites/jaas.ai
# Refactor nav to improve responsiveness and tidy up layout bugs

## Done

Migrate work carried out on Blues late last year to improve resilience and responsiveness of primary navigation. 

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- View the primary nav across breakpoints to ensure responsiveness and resilience.

## Details

Fixes https://github.com/ubuntudesign/juju-squad/issues/575
